.github/workflows/sentry_release.yml

```diff
@@ -1,12 +1,10 @@
 name: Create Sentry release
 
 on:
-  push:
-    branches:
-      - main
+  workflow_call
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.sha }}
```

Comment:
Why do we need the sha here in addition to the ref?

Reply:
What will have the same group?

Reply:
oh wait, this is the sentry_release, which is a work_flow call. we can remove these concurrency rules entirely, since this will always run under that flow.

Reply:
Same as the concurrency rules in lint-test

Reply:
yeeted here 6e9dcb7441cb19475e1c0625be6122ee754ec476